src/common/helpers/country/country-resolver.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { useStaticsQuery } from '$app/common/queries/statics';
+
+export function useCountryResolver() {
```

Comment:
We do have useResolveCountry. Can we use it?

src/common/hooks/useResolveCountry.ts